LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageProcessor.cs

```diff
@@ -486,38 +490,34 @@ private async Task<byte[]> ProcessLoraMessage(LoRaMessageWrapper loraMessage)
 
 
 
-            Logger.Log(loraDeviceInfo.DevEUI, $"processing time: {DateTime.UtcNow - startTimeProcessing}", Logger.LoggingLevel.Info);
+            Logger.Log(loraDeviceInfo!=null?loraDeviceInfo.DevEUI:devAddr, $"processing time: {DateTime.UtcNow - startTimeProcessing}", Logger.LoggingLevel.Info);
 
             return udpMsgForPktForwarder;
         }
 
-
+        private static void createRequestCache(string devAddr, LoraDeviceInfo loraDeviceInfo)

```

Comment:
Any reason to create a method with a single call? All other usages of `Cache` are done directly. Are you going to extend in the future to add more functionality?

Cosmetic: methods start with capital letter